packages/reference/src/components/EntityStatusBadge/EntityStatusBadge.tsx

```diff
@@ -34,7 +38,12 @@ export function EntityStatusBadge({
     getEntityScheduledActions,
   });
 
-  if (useLocalizedEntityStatus && activeLocales && localesStatusMap) {
+  const hasMultipleStatuses = useMemo(
+    () => entityHelpers.hasDifferentLocaleStatuses(localesStatusMap),
+    [localesStatusMap]
+  );
+
+  if (useLocalizedEntityStatus || hasMultipleStatuses) {
```

Comment:
wonder if we should have a second "feature flag" to enable the `hasMultipleStatuses` check, not that we confuse customers already with a new UX (in case the fieldStatus has different information than expected)

Reply:
Do we already have customers that have enabled and then disabled locale publishing though? The publishMode switch is still behind a FF right?

Reply:
I think we saw already something in the past when we rolled the `getEntryStatus` out without a feature flag, that the status did change for some customers -  for example if a locale was added after the entry was published